### DISCUSSION TITLE
On-premises UTMstack upgrade failure

### DISCUSSION THREAD

**Discussion by MaterialStep4** [utmstack/UTMStack#917] [Q&A]
I had a working v10 UTMStack on Ubuntu, on a local pc.  I noticed it was out of date, so I attempted to run the installer instructions.

This failed:

Welcome to Ubuntu 24.04.1 LTS (GNU/Linux 6.8.0-49-generic x86_64)

 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/pro

 System information as of Wed Dec 11 07:19:23 PM UTC 2024

  System load:  0.02               Temperature:             46.5 C
  Usage of /:   9.3% of 136.74GB   Processes:               176
  Memory usage: 5%                 Users logged in:         0
  Swap usage:   0%                 IPv4 address for enp2s0: 192.168.105.16

 * Strictly confined Kubernetes makes edge and IoT secure. Learn how MicroK8s
   just raised the bar for easy, resilient and secure K8s cluster deployment.

   https://ubuntu.com/engage/secure-kubernetes-at-the-edge

Expanded Security Maintenance for Applications is not enabled.

0 updates can be applied immediately.

Enable ESM Apps to receive additional future security updates.
See https://ubuntu.com/esm or run: sudo pro status

*** System restart required ***
Web console: https://siem:9090/ or https://192.168.105.16:9090/

Last login: Wed Dec 11 02:27:10 2024 from 192.168.105.125
asfasdf@siem:~$ sudo su
[sudo] password forasdfasdf:
root@siem:/home/asdfasdf# ./installer
### UTMStack Installer ###
Checking system requirements
Checking system requirements
Checking system requirements [OK]
Generating Stack configuration
Generating Stack configuration [OK]
Installing Stack. This may take a while.
Downloading  ghcr.io/utmstack/utmstack/bitdefender:v10
v10: Pulling from utmstack/utmstack/bitdefender
38dfbce43ec9: Already exists
40e1e1f7e51b: Pull complete
6a16ab499f13: Pull complete
4f4fb700ef54: Pull complete
Digest: sha256:57aa78e408a97378dedb19967c16cff5982f77b22264e90d584e8d05b6b326a8
Status: Downloaded newer image for ghcr.io/utmstack/utmstack/bitdefender:v10
ghcr.io/utmstack/utmstack/bitdefender:v10
Downloading  ghcr.io/utmstack/utmstack/correlation:v10
v10: Pulling from utmstack/utmstack/correlation
Digest: sha256:839ff137e5cc88ae2179a1bab17f6e0f4950207db169700085062c00102971e9
Status: Image is up to date for ghcr.io/utmstack/utmstack/correlation:v10
ghcr.io/utmstack/utmstack/correlation:v10
Downloading  ghcr.io/utmstack/utmstack/log-auth-proxy:v10
v10: Pulling from utmstack/utmstack/log-auth-proxy
7478e0ac0f23: Already exists
fa95fe7f7144: Pull complete
8120ca39f257: Pull complete
9d6680879667: Pull complete
Digest: sha256:2d95ee11961a11dbc33d77014950f38c3921150d8137c459ffe701aff230a41a
Status: Downloaded newer image for ghcr.io/utmstack/utmstack/log-auth-proxy:v10
ghcr.io/utmstack/utmstack/log-auth-proxy:v10
Downloading  utmstack.azurecr.io/logstash:v10
v10: Pulling from logstash
Digest: sha256:a91df84e34e9277a6a1610b53df488b7e48ab07496637ff7cdf5d93c8749f10f
Status: Image is up to date for utmstack.azurecr.io/logstash:v10
utmstack.azurecr.io/logstash:v10
Downloading  ghcr.io/utmstack/utmstack/office365:v10
v10: Pulling from utmstack/utmstack/office365
38dfbce43ec9: Already exists
0351a0ae7cb0: Pull complete
4b333203ea22: Pull complete
4f4fb700ef54: Pull complete
Digest: sha256:ad5ecd7884dbf0fc4bf0366a912748f5a42206cca1a7aacd6a5ba0fdb82acb1b
Status: Downloaded newer image for ghcr.io/utmstack/utmstack/office365:v10
ghcr.io/utmstack/utmstack/office365:v10
Downloading  ghcr.io/utmstack/soc-ai/soc-ai:v10
v10: Pulling from utmstack/soc-ai/soc-ai
7021d1b70935: Pull complete
8a3ccbf95df1: Pull complete
626992e82bbe: Pull complete
4f4fb700ef54: Pull complete
Digest: sha256:fa8bc66a1fc390df66ffcded0529ffc7eaa3bb858a820062c723683af8f0c2d9
Status: Downloaded newer image for ghcr.io/utmstack/soc-ai/soc-ai:v10
ghcr.io/utmstack/soc-ai/soc-ai:v10
Downloading  ghcr.io/utmstack/utmstack/frontend:v10
v10: Pulling from utmstack/utmstack/frontend
Digest: sha256:50edf0907e0a0fd60be12af23d500e764ccfc5df84da0d625cf125b21435c75f
Status: Image is up to date for ghcr.io/utmstack/utmstack/frontend:v10
ghcr.io/utmstack/utmstack/frontend:v10
Downloading  utmstack.azurecr.io/datasources:v10
Error response from daemon: manifest for utmstack.azurecr.io/datasources:v10 not                                                               found: manifest unknown: manifest tagged by "v10" is not found
exit status 1

**Comment by c3s4rfred** [comment]
Hi, @MaterialStep4, please download our last released installer and then try again the installation.

Best regards

**Comment by MaterialStep4** [comment]
I deleted the older installer files, and the new installer worked.
